tests/test_collective_ops.py

```diff
@@ -136,13 +142,18 @@ def test_send_recv_scalar_jit():
     arr = 1 * rank
     _arr = 1 * rank
 
+    @jax.jit
+    def send_jit(x):
+        Send(x, 0, tag=rank)
+        return x
```

Comment:
It's not possible to return tokens from GPU calls, so this is how a jitted version of `Send` has to be written (no more `jax.jit(lambda x: Send(x))`, because that would try to return the token).